notebook/services/kernels/kernelmanager.py

```diff
@@ -71,6 +71,16 @@ def _update_root_dir(self, proposal):
         help="""The interval (in seconds) on which to check for idle kernels exceeding the cull timeout value."""
     )
 
+    cull_connected = Bool(True, config=True,
+        help="""Whether to consider culling kernels which have one or more connections.
+        Only effective if cull_interval is not 0."""
```

Comment:
Good catch - yes, the descriptions of each should reference `cull_idle_timeout`.